rebasehelper/application.py

```diff
@@ -99,7 +100,10 @@ def __init__(self, cli_conf=None):
         self._add_report_log_file()
         if self.conf.build_tasks is None:
             self._get_spec_file()
-            self._prepare_spec_objects()
+            try:
```

Comment:
No need to catch the exception here, just let it be caught in the main() and printed to the user.